notebooks/MAST/tess_cutout_rotation_curve/tess_cutout_rotation_curve.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<p>It looks like I can't interact with the notebook in ReviewNB, but I like the use of the interactive light curve here! May also want to note that users can change the selected aperture mask in this view, or point them to the lightkurve tutorial on using <code>interact</code> with the TPFs (https://docs.lightkurve.org/tutorials/1-getting-started/interactively-inspecting-data.html).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I added this sentence: You can view the [Interactively inspecting Target Pixel Files](https://docs.lightkurve.org/tutorials/1-getting-started/interactively-inspecting-data.html) tutorial to learn about more of the features of <code>interact</code> such as how to choose a custom aperture mask or change the screen stretch.</p>